### ISSUE TITLE
Screenreaders don't announce window title when application opens.

### ISSUE THREAD

**Issue by SavagingSky** [dotnet/winforms#6896]
### .NET version

6.0

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

I haven't checked .net core, but it did the same in .net 5.0.

### Issue description

When an application loads, the form is in focus, but the active screen reader doesn't announce it's title when it should.
I have tried several workarounds, non of them make sense and actually should be done because this should work without any workarounds, but it didn't fix the issue. I have used nvda and windows10 narrator screen readers.

### Steps to reproduce

1. Create a standard windows forms app using the template without changing it.
2. Get a screen reader ([nvda](https://nvaccess.org) or windows10/11 narrator (ctrl+windows+enter) should do), launch it.
3. Build and launch.
4. The window opens and is in focus, but the screen reader doesn't announce it's title (the Form.Text property) when it pops up, but it should. To read the window title, you should alt+tab out then back again, as window title checking commands (Insert/capslock+t for nvda and capslock+t for narrator report wrong information).
I've made sure that the issue is reproducible not only on my pc, but on others as well.

**Comment by dreddy-work** [comment]
Thanks @SavagingSky for reporting this. We are working on improving the accessibility and will get this addressed in the coming release.

**Comment by Olina-Zhang** [comment]
Verified this issue on .NET 7.0.100-preview.5.22227.7 build, it is fixed:
![image](https://user-images.githubusercontent.com/26474449/165670392-279b730c-b20a-45a4-a40a-25cdda6a26a5.png)
